models/linkages/R/met2model.LINKAGES.R

```diff
@@ -18,19 +18,55 @@
 ##' @param outfolder location where model specific output is written.
 ##' @return OK if everything was succesful.
 ##' @export
-##' @author Ann Raiho
+##' @author Ann Raiho, Betsy Cowdery
 ##-------------------------------------------------------------------------------------------------#
-met2model.LINKAGES <- function(in.path, in.prefix=NULL, start.year, end.year, outfolder) {
+met2model.LINKAGES <- function(in.path, in.prefix, outfolder, start_date, end_date, ..., overwrite=FALSE,verbose=FALSE) {
    
-  out.file <- file.path(paste0(outfolder,"test_text1.txt"))
+  start_date <- as.POSIXlt(start_date, tz = "GMT")
+  end_date<- as.POSIXlt(end_date, tz = "GMT")
+  out.file <- file.path(outfolder, paste(in.prefix,
```

Comment:
In linkages, the output FOLDER can be whatever you want, but the filename needs to be test_text1.txt since the file NAME is hardcoded in the executable.